<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view_horary</class>
 <widget class="QMainWindow" name="view_horary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lbfix_Ar">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="img_H">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbfix_Ab">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>459</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionH_img">
   <property name="icon">
    <iconset resource="rsour.qrc">
     <normaloff>:/rescr/mov_obj</normaloff>:/rescr/mov_obj</iconset>
   </property>
   <property name="text">
    <string>H_img</string>
   </property>
   <property name="toolTip">
    <string>View Horary</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rsour.qrc"/>
 </resources>
 <connections/>
</ui>
